README.md

```diff
@@ -145,54 +145,21 @@ const routes = importAll.deferred('./files/*.js')
       ↓ ↓ ↓ ↓ ↓ ↓
 
 const routes = {
-  './files/a.js': function() {
+  './files/a.js': function () {
     return import('./files/a.js')
   },
-  './files/b.js': function() {
+  './files/b.js': function () {
     return import('./files/b.js')
   },
-  './files/c.js': function() {
+  './files/c.js': function () {
     return import('./files/c.js')
   },
-  './files/d.js': function() {
+  './files/d.js': function () {
     return import('./files/d.js')
   },
 }
 ```
 
-**Configure `importAll` to transform import path before generating imports**
-
```

Comment:
The docs within this area of the readme are generated. You'll want to put this below the comment that closes the section out.

Reply:
Should I update `snap-to-readme.js` to auto generate the new snapshot into README?

Reply:
Where is the `snap-to-readme.js` be run automatically?

Reply:
I found that it's troublesome to update `snap-to-readme.js` to support multiple snapshots, and include non-snapshot codes into `README.md`.

So I've manually updated it into README by your suggestion.